index.html

```diff
@@ -350,58 +331,137 @@ <h2>With COSE</h2>
         be secured using COSE [[RFC9052]] and SHOULD be identified through
         use of content types as outlined in this section.
       </p>
+
       <section>
-        <h2>Securing JSON-LD VCs with COSE</h2>
+        <h2>Securing JSON-LD Verifiable Credentials with COSE</h2>
         <p>
           This section details how to secure data with the type
           <code>application/vc+ld+json</code>
           with COSE.
         </p>
-        <p>[[RFC9052]] MAY be used to secure this media type.</p>
-        <p>When using this approach, the <code>typ</code> SHOULD be
-          <code>vc+ld+json+cose</code>.
-          See <a href="https://www.ietf.org/archive/id/draft-ietf-cose-typ-header-parameter-00.html/">I-D.ietf-cose-typ-header-parameter</a>
+        <p>
+          [[RFC9052]] MAY be used to secure this media type.
+          The <code>typ</code> header parameter SHOULD be <code>application/vc+ld+json+cose</code>.
```

Comment:
For the sake of consistency this will also need to update, depending on https://github.com/w3c/vc-jose-cose/issues/184